### ISSUE TITLE
Error on `npm test` on add-utility-docs branch

### ISSUE THREAD

**Issue by maya** [uswds/uswds-site#568]
When I run `npm test` I get this dialog popup that Chrome Canary quit unexpectedly even tho I didn't have the program open on my computer during the `node config/run-axe.js` test:

<img width="841" alt="screen shot 2018-06-08 at 10 40 06 am" src="https://user-images.githubusercontent.com/5249443/41172636-487c7562-6b09-11e8-83aa-95c7c08e0c84.png">

After I close the dialog and the test runs I get this error:
```
➜  web-design-standards-docs git:(add-utility-docs) npm test

> @ test /Users/mayaben-ari/Code/web-design-standards-docs
> npm run axe && npm run crawl && bundle exec rspec


> @ axe /Users/mayaben-ari/Code/web-design-standards-docs
> node config/run-axe.js

An error occurred during initialization.
{ Error: connect ECONNREFUSED 127.0.0.1:55018
    at Object._errnoException (util.js:1031:13)
    at _exceptionWithHostPort (util.js:1052:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1195:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 55018 }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ axe: `node config/run-axe.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ axe script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mayaben-ari/.npm/_logs/2018-06-08T17_40_26_956Z-debug.log
npm ERR! Test failed.  See above for more details.
```

cc @toolness

**Comment by toolness** [comment]
Oh, so this is happening because [chrome-launcher](https://www.npmjs.com/package/chrome-launcher) is finding Canary on your computer and running it in headless mode, and telling it to keep its remote debugging port open so that chrome-remote-interface can talk to it.

It looks like it's finding Canary, but something bad is happening and Canary's crashing, which means that its remote debugging port isn't open, and that's why the "ECONNREFUSED" error is showing up.

According to the documentation for chrome-launcher, it looks like the `CHROME_PATH` environment variable can be set to explicitly point at a binary for Chrome.  So maybe setting it to point at the binary for Chrome stable instead of having it find Canary (which could have bugs) may fix things?

try running this at your shell:

```
export CHROME_PATH="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
```

then try running `npm test` again and see what happens.  At the very least, you shouldn't see any mention of Canary crashing (although this time it might be Chrome stable that's crashing, which would be a bummer, but at least it means it's paying attention to that environment variable).

**Comment by maya** [comment]
@toolness I tried that and it still gave me the same Canary error dialog.

**Comment by maya** [comment]
Also, I deleted Canary from my machine (no more dialog, yay!) now I'm getting this error:
```
> @ axe /Users/mayaben-ari/Code/web-design-standards-docs
> node config/run-axe.js

Static file server is listening at http://f09r9j-2jmg3qt:52929.
Chrome is debuggable on http://localhost:52930.
Running aXe on:
  /page-templates/docs/ A network request failed to load.
{ requestId: '25649B1703C9FE22EEA52709C78A9CEC',
  timestamp: 509126.094402,
  type: 'Document',
  errorText: 'net::ERR_NAME_NOT_RESOLVED',
  canceled: false }
Terminating with exit code 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ axe: `node config/run-axe.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ axe script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mayaben-ari/.npm/_logs/2018-06-08T20_53_59_941Z-debug.log
npm ERR! Test failed.  See above for more details.
```

**Comment by maya** [comment]
Visiting the page `/page-templates/docs/`, I see this in the console:

`:4000/favicon.ico:1 - Failed to load resource: the server responded with a status of 404 (Not Found)`, so I'm wondering if that's the resource is failing to load.

**Comment by maya** [comment]
Fixed in #570.
